_board/CC03.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: download
-board_id: "cc03"
-title: "CC03 Download"
-name: "CC03"
+board_id: "XinaBox CC03
```

Comment:
no, match the *folder name* https://github.com/adafruit/circuitpython/tree/master/ports/atmel-samd/boards/xinabox_cc03